src/modules/Instabug.ts

```diff
@@ -46,6 +46,11 @@ export {
   strings,
 };
 
+interface InstabugConfig {
```

Comment:
**suggestion:** I think we should export that type to the users, what do you think?

Reply:
Makes sense!